tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -66,6 +78,12 @@ spec:
             exit 1
         fi
 
+        prefixedTag=""
+        if [ -n "$(params.tagPrefix)" ]; then
+            prefixedTag="$(params.tagPrefix)-$(date "+${params.timestampFormat}")"
+            echo -en "${prefixedTag}" > $(results.commonTag.path)
```

Comment:
Nit: The `-e` option for echo is not needed here. We only need `-n`.

Reply:
Thanks, the echo is correct now. But for the comment below - I'm actually not sure what happened here, but I see my comment both below (without a thread) and above (in the thread: https://github.com/redhat-appstudio/release-service-catalog/pull/223#discussion_r1352565783 ) And I was commenting on the new version that is still current now.